flask/cli.py

```diff
@@ -341,43 +341,40 @@ def _load_plugin_commands(self):
 
     def get_command(self, ctx, name):
         self._load_plugin_commands()
-
-        # We load built-in commands first as these should always be the
```

Comment:
Custom commands through entry points can already override the built-ins. https://github.com/ei-grad/flask-shell-ipython/blob/3f8a82013a6fd8ef42156470e30e3638604190a1/setup.py#L17